arangod/VocBase/vocbase.cpp

```diff
@@ -1283,6 +1284,39 @@ std::shared_ptr<arangodb::LogicalCollection> TRI_vocbase_t::createCollection(
   }
 }
 
+ResultT<std::vector<std::shared_ptr<arangodb::LogicalCollection>>>
+TRI_vocbase_t::createCollections(
+    std::vector<arangodb::PlanCollection> const& collections,
+    bool allowEnterpriseCollectionsOnSingleServer) {
+  // TODO: Need to get rid of this collection. Distribute Shards like
+  // is now denoted inside the PlanCollection
+  std::shared_ptr<LogicalCollection> colToDistributeShardsLike;
```

Comment:
Is this TODO still relevant? The variable seems to be unused at least.

Reply:
Yes, will be tackled in follow up PR.